index.html

```diff
@@ -1,12 +1,262 @@
+              разочаруетесь!</p>
+            <svg xmlns="http://www.w3.org/2000/svg" width="76" height="72" fill="none">
+              <path fill="#83B3D3"
+                d="M71.58 47.7H5.6c-2.3 0-4.11 1.9-4.11 4.3v3.5c0 2.4 1.9 4.3 4.11 4.3h.9v1.1c0 5.5 4.31 10.1 9.63 10.1h44.7c5.32 0 9.63-4.5 9.63-10.1v-1.1h.9c2.31 0 4.12-1.9 4.12-4.3V52a4 4 0 0 0-3.91-4.3Zm-3.91 13.2c0 3.9-3.01 7-6.72 7H16.14c-3.71 0-6.72-3.2-6.72-7v-1.1h58.25v1.1Zm5.1-5.4c0 .7-.6 1.3-1.2 1.3H5.52c-.7 0-1.2-.6-1.2-1.3V52c0-.7.6-1.3 1.2-1.3h65.97c.7 0 1.2.6 1.2 1.3.1 0 .1 3.5.1 3.5ZM17.14 39.4l5.41 4.1 5.32-4.1 5.3 4.1 5.32-4.1 5.42 4.1 5.31-4.1 5.31 4.1 5.42-4.1 5.31 4.1 6.22-4.7-1.7-2.5-4.52 3.4-5.31-4.1-5.42 4.1-5.31-4.1-5.31 4.1-5.42-4.1-5.31 4.1-5.31-4.1-5.32 4.1-5.41-4.1-5.31 4.1-4.52-3.4-1.7 2.5 6.22 4.7 5.3-4.1ZM9.52 32.5h60.95c0-.6.1-1.2.1-1.9v-1.2C69.87 13.7 55.84 1 38.6 1 21.25 1 7.21 13.7 6.51 29.5v1.2c0 .6 0 1.2.1 1.8h2.91ZM38.59 4c15.64 0 28.47 11.3 29.08 25.5H9.52C10.12 15.4 22.95 4 38.59 4Z" />
+              <path fill="#83B3D3"
+                d="M38.6 18.3c1.05 0 1.9-.9 1.9-2s-.85-2-1.9-2c-1.06 0-1.91.9-1.91 2s.85 2 1.9 2ZM30.77 13.2c1.06 0 1.9-.9 1.9-2s-.84-2-1.9-2c-1.05 0-1.9.9-1.9 2s.85 2 1.9 2ZM46.31 13.2c1.05 0 1.9-.9 1.9-2s-.85-2-1.9-2c-1.05 0-1.9.9-1.9 2s.85 2 1.9 2Z" />
+            </svg>
+          </li>
+
+          <!--3-->
+          <li class="recommendations-item recommendations-item-opaque">
+            <h3 class="recommendations-item-title">Сувениры</h3>
+            <p class="recommendations-item-content">Не только китайского,<br>но и настоящего местного<br>производства!
+            </p>
+            <svg xmlns="http://www.w3.org/2000/svg" width="65" height="76" fill="none">
+              <path fill="#83B3D3"
+                d="M61.6 22.46H36.8c13.8-6.9 14.9-11.3 14.4-13.7-1.2-4.01-5.7-7.01-10-5.41-5.6 2.1-7.1 10.4-8.6 16.41-2-6.1-3.8-13.8-8.2-17.71-1.3-1.1-3.1-2.3-5.8-2-4 .4-9.8 5.4-7.5 11 2.3 5.4 9.7 8.81 14.6 11.51H3.4a2.9 2.9 0 0 0-2.9 2.9v11.31c0 1.6 1.3 2.9 2.9 2.9h.3V76h57.8V39.57h.1c1.6 0 2.9-1.3 2.9-2.9v-11.3c0-1.6-1.3-2.9-2.9-2.9ZM42.9 5.86c2.1-.3 5 1.3 5.3 3.2.4 2.8-3.4 5.3-5.1 6.4-2.5 1.7-4.9 2.6-7.3 4.1.9-3.7 2.5-13 7.1-13.7Zm-22.9 10c-2.2-1.5-6.5-4.4-6.4-7.8.1-3.61 5.7-6.61 8.8-3.91 3.9 3.4 5.7 11.1 7.1 16.61-3.2-1.5-6.3-2.8-9.5-4.9ZM26 73H6.6V39.57H26V73Zm0-36.53H3.5v-11H26v11Zm10.2 0V73H29V25.57h7.2v10.9ZM58.5 73H39.1V39.57h19.4V73Zm2.9-36.53H39.1v-11h22.3v11Z" />
+            </svg>
+          </li>
+
+        </ul>
+      </section>
+
+      <!-- Заинтересовались? -->
+      <section class="interested" data-test="search">
+        <h2 class="interested-title">Заинтересовались?</h2>
+        <p class="interested-content">Укажите предполагаемые даты поездки,<br>и мы покажем вам лучшие предложения
+          гостиниц в Седоне</p>
+        <button class="button interested-button" type="button">Поиск гостиницы в Седоне</button>
```

Comment:
```suggestion
        <a class="button interested-button" href="#">Поиск гостиницы в Седоне</a>
```
Это ссылка, которая куда-то ведет.
Может, на вторую страницу, может к форме на этой. Мы не знаем.